lib/src/widgets/chat/ds_file_message_bubble.widget.dart

```diff
@@ -60,14 +63,27 @@ class DSFileMessageBubble extends StatelessWidget {
           padding: EdgeInsets.zero,
           align: align,
           style: style,
-          child: SizedBox(
-            height: 80.0,
-            child: Row(
-              mainAxisSize: MainAxisSize.min,
-              children: [
-                _buildIcon(),
-                _buildText(),
-              ],
+          child: Padding(
+            padding: const EdgeInsets.all(12.0),
+            child: LayoutBuilder(
```

Comment:
I think that this ```LayoutBuilder``` could wrap just the ```DSShowMoreText``` widget.

Reply:
Done!